### DISCUSSION TITLE
Remove the log

### DISCUSSION THREAD

**Discussion by AlABarazi** [mesa/mesa#1337] [Q&A] [answered]
Hi,
When runninig the simulation I am getting these in terminal  
{"type":"get_step","step":4}
how to turn this off ? 
Thanks!

**Comment by rht** [comment, marked as answer]
After the `server = ModularServer(...)` initialization, you can do `server.verbose = False`. This is not available in the `ModularServer` `__init__` args, but should have been. If you want, you can make a PR to add this arg.
